crypto/fipsmodule/rsa/rsa.c

```diff
@@ -551,12 +528,9 @@ int RSA_add_pkcs1_prefix(uint8_t **out_msg, size_t *out_msg_len,
 }
 
 int rsa_sign_no_self_test(int hash_nid, const uint8_t *digest,
-                          size_t digest_len, uint8_t *out, unsigned *out_len,
+                          unsigned digest_len, uint8_t *out, unsigned *out_len,
                           RSA *rsa) {
   if (rsa->meth->sign) {
-    if (!rsa_check_digest_size(hash_nid, digest_len)) {
-      return 0;
-    }
     // All supported digest lengths fit in |unsigned|.
     assert(digest_len <= EVP_MAX_MD_SIZE);
     OPENSSL_STATIC_ASSERT(EVP_MAX_MD_SIZE <= UINT_MAX, digest_too_long);
```

Comment:
I think this whole if block needs to be removed in this revert, you might have resolved the conflict in the file the wrong way.

Reply:
Done, CI running now...